src/xwayland.rs

```diff
@@ -670,4 +672,17 @@ impl Primary for XWayland {
             vec![1],
         )
     }
+
+    fn set_mode_control(
+        &self,
+        width: u32,
+        height: u32,
+        super_res: u32
+    ) -> Result<(), Box<dyn std::error::Error>> {
+        self.set_xprop(
+            self.root_window_id,
+            GamescopeAtom::ModeControl,
+            vec![self.root_window_id, width, height, super_res]
+        )
+    }
```

Comment:
It looks like this also requires the xwayland id as an argument.

E.g. doing `set_mode_control(0, 800, 600, 0)` will set the resolution on the first xwayland instance (where the overlay runs) and doing `set_mode_control(1, 800, 600, 0)` will set the resolution on the second xwayland instance (where the game is running).

```suggestion
    fn set_mode_control(
        &self,
        xwayland_id: u32,
        width: u32,
        height: u32,
        super_res: u32,
    ) -> Result<(), Box<dyn std::error::Error>> {
        self.set_xprop(
            self.root_window_id,
            GamescopeAtom::ModeControl,
            vec![xwayland_id, width, height, super_res],
        )
    }
```